charts/sysdig-deploy/README.md

```diff
@@ -30,7 +30,7 @@ Currently included components:
 2. Collect the following values:
 
    - ACCESS_KEY: This is your Sysdig access key
-   - SAAS_REGION: The Sysdig SAAS region the agents will connect to. See [Regions and IP Ranges](https://docs.sysdig.com/en/docs/administration/saas-regions-and-ip-ranges/) for more information. The Collector URL is custom for on-prem installations.
+   - SAAS_REGION: The Sysdig SAAS region the agents will connect to. Possible values are `"us1"`, `"us2"`, `"us3"`, `"us4"`, `"eu1"`, `"au1"`, and `"custom"`. For on-prem installations, use `custom` and override the endpoints for each component (see [Configuration](#configuration) below).
```

Comment:
- SAAS_REGION: The Sysdig SAAS region that the agents will connect to.  Use one of the following values:
   -  `"us1"`
   - `"us2"`
   - `"us3"`
   - `"us4"`
   - `"eu1"`, 
   - `"au1"`
   -  `"custom"`:  For on-prem installations, use `custom` and override the endpoints for each component.
 For more information, see [Configuration](#configuration).